R/conversion.R

```diff
@@ -294,3 +316,18 @@ py_to_r.pandas.core.frame.DataFrame <- function(x) {
   df
   
 }
+
+#' @export
+summary.pandas.core.frame.DataFrame <- function(object, ...) {
+  summary.pandas.core.series.Series(object, ...)
+}
+
+#' @export
+length.pandas.core.frame.DataFrame <- function(x) {
+  length.pandas.core.series.Series(x)
+}
+
+#' @export
+dim.pandas.core.frame.DataFrame <- function(x) {
+  dim.pandas.core.series.Series(x)
```

Comment:
Similar comment here (have a helper `py_shape()` function that these `dim` methods both call)